src/components/CreateNewProjectWizard/CreateNewProjectWizard.js

```diff
@@ -96,16 +99,46 @@ class CreateNewProjectWizard extends PureComponent<Props, State> {
     }
   };
 
+  checkProjectLocationUsage = () => {
+    return new Promise((resolve, reject) => {
+      const projectName = getProjectNameSlug(this.state.projectName);
+      if (checkIfProjectExists(this.props.projectHomePath, projectName)) {
+        // show warning that the project folder already exists & stop creation
+        dialog.showMessageBox(
+          {
+            type: 'warning',
+            title: 'Project directory exists',
+            message:
+              'Please pick a different location or name.\nYou could also manually delete the project from disk and create the project with that name.',
+            buttons: ['OK'],
+          },
+          result => {
+            if (result === 0) {
+              return reject();
+            }
+
+            resolve();
+          }
+        );
+      } else {
+        resolve();
+      }
+    });
+  };
   handleSubmit = () => {
     const currentStepIndex = FORM_STEPS.indexOf(this.state.currentStep);
     const nextStep = FORM_STEPS[currentStepIndex + 1];
 
     if (!nextStep) {
-      this.setState({
-        activeField: null,
-        status: 'building-project',
-      });
-      return;
+      return this.checkProjectLocationUsage()
+        .then(() => {
+          // not in use
+          this.setState({
+            activeField: null,
+            status: 'building-project',
+          });
+        })
+        .catch(() => {});
```

Comment:
OK, added the comment as I think the code is easier to understand. 

Not sure how to code if the promise will reject the error and then we're showing the dialog it would require another promise because of the callback.